### PR TITLE
Merge/tasks subtasks

### PR DESCRIPTION
**Pull request by maxahn**
Things are merged and semi working, i'll have to fix the task and subtask creation actions broken during this merge but i'll do it after dinner  and I wante everyone caught up :/

### CODE REVIEW

File: src/components/Feed/components/TaskUnit/components/SocialUnit/actions.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export const addClap = (info) => {
```
Comment (AamirL1011) [inline src/components/Feed/components/TaskUnit/components/SocialUnit/actions.js:1]: I deleted all the action files that are outside of the actions folder.

File: src/components/Feed/components/TaskUnit/components/SocialUnit/components/MangoPopover/actions.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const addMango = (info) => {
```
Comment (AamirL1011) [inline src/components/Feed/components/TaskUnit/components/SocialUnit/components/MangoPopover/actions.js:1]: As mentioned in my first comment, this file can be deleted.

File: src/reducers/feedReducer.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
+const feedState = {
```
Comment (AamirL1011) [inline src/reducers/feedReducer.js:1]: This file should be deleted

File: src/reducers/userProfileReducer.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
+const userProfileState = {
```
Comment (AamirL1011) [inline src/reducers/userProfileReducer.js:1]: This file should be deleted.

File: src/components/SubTask/components/SubTaskList/components/subTaskItem/index.js [hunk 5]
```diff
@@ -100,10 +111,10 @@ class SubTaskItem extends Component {
                 role="button"
                 tabIndex={0}
                 onClick={() => {
-                  this.changeState(index, subTask, task, tasks);
+                  this.changeState(subTask, task, tasks);
                 }}
                 onKeyDown={() => {
-                  this.changeState(index, subTask, task, tasks);
+                  this.changeState(subTask, task, tasks);
```
Comment (MS642) [inline src/components/SubTask/components/SubTaskList/components/subTaskItem/index.js:117]: Could you revert these changes? I added those to get rid of calculating the subtask index, and fix issues with having same subtask name
